depfinder/stdliblist.py

```diff
@@ -0,0 +1,11 @@
+import sys
+
+
+try:
+    from stdlib_list import stdlib_list
+    pyver = '%s.%s' % (sys.version_info.major, sys.version_info.minor)
+    builtin_modules = stdlib_list(pyver)
+    del pyver
+except ImportError:
+    # assuming py>=3.10
```

Comment:
This *should* be the case now, but if stdlib-list was somehow missing in Python <=3.9, then error message could be a bit confusing?

Reply:
It is listed as a dependency for py<3.10, so the only way that scenario would happen is in a broken installation with "no-deps"  and people doing that are usually experts. The error message would be:

```
AttributeError: module 'sys' has no attribute 'stdlib_module_names'
```

That is, IMO, the correct information an advanced users should know but I accept suggestions for a better one.

Reply:
Yeah, I suppose you are right, that makes sense.

I guess I would lean towards switching the `try` and `except` parts or doing a `sys.version_info` check.

And I know you're going another way here, but I also feel like it could be useful to have this in stdlib-list, e.g. to support listing the 3.10 stdlib from an older Python version, and to reduce the number of times this code here would have to be written perhaps.

Reply:
Yep. It makes sense to put in stliblib-list. I completely forgot I am a maintainer there and I can port this over tomorrow.